package.json

```diff
@@ -8,7 +8,7 @@
     "clean": "rm -rf build",
     "prepublish": "npm run build && npm run lint",
     "precommit": "npm run lint && npm run build && npm run test",
-    "lint": "resin-lint --typescript 1 lib/*.ts typings/*.ts test/*.ts",
+    "lint": "resin-lint --typescript lib/*.ts typings/*.ts test/*.ts",
```

Comment:
That number "1" there -- it looked like it was accidentally typed, because it was causing lint failure:
```
Unhandled rejection TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be one of type string, Buffer, or URL. Received type number
```
Or else, what's it meant to be?

Reply:
@pdcastro in an old version of `resin-lint`, `--typescript` was a flag that required a parameter.